CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+##2015-04-15 - Supported Release 1.0.0
+###Summary
+Add ability to reboot linux distro's as well as Windows.
+
+###Features
+* Add linux support
+* Remove prompt for windows reboot
+
```

Comment:
Make sure to add a line about the `catalog_apply_timeout` parameter being removed.